README.md

```diff
@@ -1,14 +1,25 @@
 # OpenTelemetry Lambda
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(Java)?label=CI%20%28Java%29&style=for-the-badge)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(Collector)?label=CI%20%28Collector%29&style=for-the-badge)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(NodeJS)?label=CI%20%28NodeJS%29&style=for-the-badge)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/open-telemetry/opentelemetry-lambda/Continuous%20Build%20(Terraform)?label=CI%20%28Terraform%29&style=for-the-badge)
 
+## [OpenTelemetry Collector extension](collector/README.md)
+The OpenTelemetry Collector Lambda Extension provides a mechanism to export telemetry aynchronously from AWS Lambdas. It does this by embedding a stripped-down version of [OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) inside an [AWS Extension Layer](https://aws.amazon.com/blogs/compute/introducing-aws-lambda-extensions-in-preview/). This allows lambdas to use the OpenTelemetry Collector Exporter to send traces and metrics to any configured backend.
 
-### [Design Proposal](docs/design_proposal.md)
+## Lambda Language Support
 
-### [OpenTelemetry Collector extension](collector/README.md)
+* ### [Python](python/README.md)
+* ### [Java](java/README.md)
+* ### [NodeJS](nodejs/README.md)
+* ### [.Net](dotnet/README.md)
 
-### [OpenTelemetry Python Lambda support](python/README.md) 
 
-### [OpenTelemetry Java Lambda support](java/README.md)
+## FAQ
+* What exporters/recievers/processors are included from the OpenTelemetry Collector?
+    * For a full list of currently included components, see [here](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/collector/lambdacomponents/default.go#L18).
+* Is the Lambda layer provided or do I need to build it and distribute it myself?
+    * The layer is currently not provided and will need to be built manually. For more information see [here](collector/README.md)
```

Comment:
```suggestion
> What exporters/recievers/processors are included from the OpenTelemetry Collector?
For a full list of currently included components, see [here](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/collector/lambdacomponents/default.go#L18).
> Is the Lambda layer provided or do I need to build it and distribute it myself?
This repository does not provide pre-build Lambda layers. They must be built manually and saved in your AWS account. This repo has files to facilitate doing that. More information is provided in [the Collector folder's README](collector/README.md).
```

Reply:
LGTM. I'll just make some small adjustments to keep formatting correct.